ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.14__jenkins-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,60 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.19-openshift-4.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.13
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |-
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13
+    COPY . .
+    RUN chgrp -R 0 /go && \
+    chmod -R g=u /go
+    RUN mkdir -p /.cache
+    RUN chgrp -R 0 /.cache && \
+    chmod -R g=u /.cache
+    RUN chgrp -R 0 /var && \
```

Comment:
Looks like the permissions change on `/var` is missing in the Hypershift version of this image. Not sure if this is on purpose or if it needs to be put into the other image.

We may want to consider just writing a permanent Dockerfile for this so we don't have to maintain multiple files with `dockerfile_literal`.